tasks/webfont.js

```diff
@@ -310,7 +325,28 @@ module.exports = function(grunt) {
 		 *
 		 * @param {Function} done
 		 */
-		function generateStylesheet(done) {
+		function generateStylesheets(done) {
+			// Convert codepoints to array of strings
+			var codepoints = [];
+			_.each(o.glyphs, function(name) {
+				codepoints.push(o.codepoints[name].toString(16));
+			});
+			o.codepoints = codepoints;
+
+			// Prepage glyph names to use as CSS classes
+			o.glyphs = _.map(o.glyphs, classnameize);
+
+			o.stylesheets.forEach(generateStylesheet);
+
+			done();
```

Comment:
Yes, all the code is synchronous and it ends with a `fs.writeFileSync` so it waits for the file to be written before continuing (same as before this PR).